docs/src/main/asciidoc/spring-integration-pubsub.adoc

```diff
@@ -71,28 +71,21 @@ Then, we declare a `PubSubInboundChannelAdapter` bean.
 It requires the channel we just created and a `SubscriberFactory`, which creates `Subscriber` objects from the Google Cloud Java Client for Pub/Sub.
 The Spring Boot starter for GCP Pub/Sub provides a configured `SubscriberFactory`.
 
-The `PubSubInboundChannelAdapter` supports three acknowledgement modes, with `AckMode.AUTO` being the default value;
+The `PubSubInboundChannelAdapter` supports three acknowledgement modes, with `AckMode.AUTO` being the default value:
```

Comment:
Maybe put a header above this sentence.
Something like: "Acknowledging messages and handling failures"